modules/datastore/src/DataDictionary/AlterTableQuery/MySQLQuery.php

```diff
@@ -92,6 +95,31 @@ class MySQLQuery extends AlterTableQueryBase implements AlterTableQueryInterface
     'duration' => 'TINYTEXT',
   ];
 
+  /**
+   * ConfigFactory object.
+   *
+   * @var \Drupal\Core\Config\ConfigFactoryInterface
+   */
+  protected ConfigFactoryInterface $configFactory;
+
+  /**
+   * Api constructor.
+   */
+  public function __construct(
+    ConfigFactoryInterface $configFactory
+  ) {
+    $this->configFactory = $configFactory;
+  }
```

Comment:
@janette because you are introducing a constructor here, you have to also handle calling the parent constructor for `AlterTableQueryBase` which is what the test does on line https://github.com/GetDKAN/dkan/pull/4261/files#diff-0136d115ea55175e8fb916adbd8e0e9de87e7a713fc6a31d0d1852bd3753c0e9R101

Reply:
One option could be moving the config factory to the base class which might be useful but could have downstream effects.